tritium-metrics-jvm/src/main/java/com/palantir/tritium/metrics/jvm/JvmMetrics.java

```diff
@@ -17,13 +17,13 @@
 package com.palantir.tritium.metrics.jvm;
 
 import com.codahale.metrics.jvm.BufferPoolMetricSet;
-import com.codahale.metrics.jvm.ClassLoadingGaugeSet;
 import com.codahale.metrics.jvm.MemoryUsageGaugeSet;
 import com.codahale.metrics.jvm.ThreadStatesGaugeSet;
```

Comment:
Just curious, @carterkozak are you planning to replace all these gauge sets with metric-schema?

Reply:
Yep, just slowly pushing through them between other things